app/routers/agenda.py

```diff
@@ -39,6 +40,21 @@ def agenda(
     """Route for the agenda page, using dates range or exact amount of days."""
 
     user_id = 1  # there is no user session yet, so I use user id- 1.
+
+    get_a_week_int = 6
```

Comment:
why the `get` prefix? Also, it looks like a constant, consider changing it to ALL_CAPITAL

Reply:
done